File: UnicoVehicle/UnicoVehicle.DTO/PostBuyingDetail.cs
using System;
namespace UnicoVehicle.DTO
{
    public class PostBuyingDetail
    {
        public int PostBuyingDetailId { get; set; }
        public int OrderId { get; set; }
        public Miscellaneous.InsuranceCompany InsuranceCompany { get; set; }
        public InsuranceType InsuranceType { get; set; }
        public double LoanEMI { get; set; }
        public double InsurancePremium { get; set; }
        public DateTime TaxValidity { get; set; }
        public DateTime InsuranceValidity { get; set; }
        public int FreeService { get; set; }
        public bool PaymentReceived { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/PostBuyingDetail.dart
import './Miscellaneous/InsuranceCompany.dart';
import './InsuranceType.dart';

class PostBuyingDetail {
  final int postBuyingDetailId;
  final int orderId;
  final InsuranceCompany insuranceCompany;
  final InsuranceType insuranceType;
  final double loanEMI;
  final double insurancePremium;
  final DateTime taxValidity;
  final DateTime insuranceValidity;
  final int freeService;
  final bool paymentReceived;

  PostBuyingDetail({
    this.freeService,
    this.insuranceCompany,
    this.insurancePremium,
    this.insuranceType,
    this.insuranceValidity,
    this.loanEMI,
    this.orderId,
    this.paymentReceived,
    this.postBuyingDetailId,
    this.taxValidity,
  });
}
